### PR TITLE
Fix Editable Tags Modal

### PR DESCRIPTION
**Pull request by zorazrr**
## Summary

Tiny fix of #173 and #168 

## Changes

- Fetch team and interest after the modal is closed so that changes are not saved (without clicking save) on the frontend when reopening the modal.

### CODE REVIEW

File: client/src/components/Modals/EditableTags/EditableTag.tsx [hunk 1]
```diff
@@ -56,7 +58,7 @@ const EditableTagModal: FC<EditableTagModalProps> = ({
     }));
 
     setClonedTags([...clone]);
-
+    onFetch();
     // eslint-disable-next-line react-hooks/exhaustive-deps
```
Comment (amit-sawhney) [inline client/src/components/Modals/EditableTags/EditableTag.tsx:62]: This can be removed

**Review by amit-sawhney — APPROVED** [review]
Great fix!

File: client/src/components/Modals/EditableTags/EditableTag.tsx (hunk 1 above)
Comment (amit-sawhney) [reply client/src/components/Modals/EditableTags/EditableTag.tsx:62]: wait this is not the line I clicked oops. I meant line 62

Comment (amit-sawhney) [inline client/src/components/Modals/EditableTags/EditableTag.tsx:62]: This is what I meant can be removed.

Comment (amit-sawhney) [reply client/src/components/Modals/EditableTags/EditableTag.tsx:62]: bruh. why does it choose the wrong line. line 62. the eslint-disable.

Comment (zorazrr) [reply client/src/components/Modals/EditableTags/EditableTag.tsx:62]: Ye but removing it brings back the lint warning `React Hook useEffect has missing dependencies: 'allTags' and 'onFetch'. Either include them or remove the dependency array. If 'onFetch' changes too often, find the parent component that defines it and wrap that definition in useCallback`

**Review by AndrewLester — APPROVED** [review]
Looks good

File: client/src/components/Modals/EditableTags/EditableTag.tsx (hunk 1 above)
Comment (amit-sawhney) [reply client/src/components/Modals/EditableTags/EditableTag.tsx:62]: These should have been addressed before merging.

**Pull request merged**